# INSTALL_LuMP_Pulsar_TEMPO2_P0.patch
# patch file 0 for TEMPO2
# 2014 Oct 06  James Anderson  --- GFZ start
#
# Copyright (c) 2014 James M. Anderson <anderson@gfz-potsdam.de>
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
#
#
# Note that the SOFA software in this patch is presumably covered by a different
# copyright.  See http://sourceforge.net/projects/tempo2/
# for more information about TEMPO2 copyright issues.
#
################################################################################
diff -Naur tempo2-2013.9.1/T2runtime/observatory/observatories.dat tempo2-2013.9.1_new/T2runtime/observatory/observatories.dat
--- tempo2-2013.9.1/T2runtime/observatory/observatories.dat	2013-08-20 17:13:57.000000000 +0200
+++ tempo2-2013.9.1_new/T2runtime/observatory/observatories.dat	2014-10-06 19:38:14.084513790 +0200
@@ -25,9 +25,42 @@
  5085442.780   2668263.483     -2768697.034      Hartebeesthoek      hart
  3828445.659   445223.600000    5064921.5677     WSRT                wsrt
  3826577.462   461022.624       5064892.526      LOFAR               lofar
- 4324017.054   165545.160       4670271.072      FR606               FRlfr
- 4034100.228   487013.084       4900229.363      DE601               EFlfr
- 4008462.280   -100376.948      4943716.600      UK608               UKlfr
+ 4034038.635    487026.223      4900280.057      DE601LBA            EFlfrlba
+ 4034038.635    487026.223      4900280.057      DE601LBH            EFlfrlbh
+ 4034101.901    487012.401      4900230.210      DE601HBA            EFlfrhba
+ 4034101.901    487012.401      4900230.210      DE601               EFlfr
+ 4152561.068    828868.725      4754356.878      DE602LBA            UWlfrlba
+ 4152561.068    828868.725      4754356.878      DE602LBH            UWlfrlbh
+ 4152568.416    828788.802      4754361.926      DE602HBA            UWlfrhba
+ 4152568.416    828788.802      4754361.926      DE602               UWlfr
+ 3940285.328    816802.001      4932392.757      DE603LBA            TBlfrlba
+ 3940285.328    816802.001      4932392.757      DE603LBH            TBlfrlbh
+ 3940296.126    816722.532      4932394.152      DE603HBA            TBlfrhba
+ 3940296.126    816722.532      4932394.152      DE603               TBlfr
+ 3796327.609    877591.315      5032757.252      DE604LBA            POlfrlba
+ 3796327.609    877591.315      5032757.252      DE604LBH            POlfrlbh
+ 3796380.254    877613.809      5032712.272      DE604HBA            POlfrhba
+ 3796380.254    877613.809      5032712.272      DE604               POlfr
+ 4005681.742    450968.282      4926457.670      DE605LBA            JUlfrlba
+ 4005681.742    450968.282      4926457.670      DE605LBH            JUlfrlbh
+ 4005681.407    450968.304      4926457.940      DE605HBA            JUlfrhba
+ 4005681.407    450968.304      4926457.940      DE605               JUlfr
+ 4323980.155    165608.408      4670302.803      FR606LBA            FRlfrlba
+ 4323980.155    165608.408      4670302.803      FR606LBH            FRlfrlbh
+ 4324017.054    165545.160      4670271.072      FR606HBA            FRlfrhba
+ 4324017.054    165545.160      4670271.072      FR606               FRlfr
+ 3370287.366    712053.586      5349991.228      SE607LBA            ONlfrlba
+ 3370287.366    712053.586      5349991.228      SE607LBH            ONlfrlbh
+ 3370272.092    712125.596      5349990.934      SE607HBA            ONlfrhba
+ 3370272.092    712125.596      5349990.934      SE607               ONlfr
+ 4008438.796    -100310.064     4943735.554      UK608LBA            UKlfrlba
+ 4008438.796    -100310.064     4943735.554      UK608LBH            UKlfrlbh
+ 4008462.280    -100376.948     4943716.600      UK608HBA            UKlfrhba
+ 4008462.280    -100376.948     4943716.600      UK608               UKlfr
+ 2136833.225    810088.740      5935285.279      FI609LBA            Filfrlba
+ 2136833.225    810088.740      5935285.279      FI609LBH            Filfrlbh
+ 2136819.1940   810039.5757     5935299.0536     FI609HBA            Filfrhba
+ 2136819.1940   810039.5757     5935299.0536     FI609               Filfr
  3307865.236   2487350.541      4836939.784      UTR-2               UTR2
   1656318.94    5797865.99      2073213.72       GMRT                gmrt
 #
